src/templates/en/2021/methodology.html

```diff
@@ -0,0 +1,485 @@
+      </p>
+
+      <p>
+        HTTP Archive runs the latest version of Wappalyzer for all web pages. As of July {{ year }} the Web Almanac used the <a hreflang="en" href="https://github.com/AliasIO/Wappalyzer/releases/tag/v6.7.7">6.7.7 version</a> of Wappalyzer.
+      </p>
+
+      <p>
+        Wappalyzer powers many chapters that analyze the popularity of developer tools like WordPress, Bootstrap, and jQuery. For example, the <a href="{{ url_for('chapter', year=year, lang=lang, chapter='ecommerce') }}">Ecommerce</a> and <a href="{{ url_for('chapter', year=year, lang=lang, chapter='cms') }}">CMS</a> chapters rely heavily on the respective <a hreflang="en" href="https://www.wappalyzer.com/categories/ecommerce">Ecommerce</a> and <a hreflang="en" href="https://www.wappalyzer.com/categories/cms">CMS</a> categories of technologies detected by Wappalyzer.
+      </p>
+
+      <p>
+        All detection tools, including Wappalyzer, have their limitations. The validity of their results will always depend on how accurate their detection mechanisms are. The Web Almanac will add a note in every chapter where Wappalyzer is used but its analysis may not be accurate due to a specific reason.
+      </p>
+    </section>
+
+    <section>
+      <h3 id="chrome-ux-report"><a href="#chrome-ux-report" class="anchor-link">Chrome UX Report</a></h3>
+
+      <p>
+        The <a hreflang="en" href="https://developers.google.com/web/tools/chrome-user-experience-report">Chrome UX Report</a> is a public dataset of real-world Chrome user experiences. Experiences are grouped by websites&#8217; origin, for example <code>https://www.example.com</code>. The dataset includes distributions of UX metrics like paint, load, interaction, and layout stability. In addition to grouping by month, experiences may also be sliced by dimensions like country-level geography, form factor (desktop, phone, tablet), and effective connection type (4G, 3G, etc.).
+      </p>
+
+      <p>
+        As of this year, the Chrome UX Report dataset now includes relative <a href="https://developers.google.com/web/updates/2021/03/crux-rank-magnitude">website ranking data</a>. These are referred to as <em>rank magnitudes</em> because, as opposed to fine-grained ranks like the #1 or #116 most popular websites, websites are grouped into rank buckets from the top 1k, top 10k, up to the top 10M. Each website is ranked according to the number of <a href="https://developers.google.com/web/tools/chrome-user-experience-report/#methodology">eligible</a> page views on all of its pages combined. This year's Web Almanac makes extensive use of this new data as a way to explore variations in the way the web is built by site popularity.
```

Comment:
Is 1k, 10k a bit colloquial?

```suggestion
        As of this year, the Chrome UX Report dataset now includes relative <a href="https://developers.google.com/web/updates/2021/03/crux-rank-magnitude">website ranking data</a>. These are referred to as <em>rank magnitudes</em> because, as opposed to fine-grained ranks like the #1 or #116 most popular websites, websites are grouped into rank buckets from the top 1,000, top 10,000, up to the top 10 million. Each website is ranked according to the number of <a href="https://developers.google.com/web/tools/chrome-user-experience-report/#methodology">eligible</a> page views on all of its pages combined. This year's Web Almanac makes extensive use of this new data as a way to explore variations in the way the web is built by site popularity.
```

Also worth expanding on not every bucket is full (particularly the last bucket). Or TMI?

Reply:
> Is 1k, 10k a bit colloquial?

This notation is consistent with the way it's described in the [source doc](https://developers.google.com/web/updates/2021/03/crux-rank-magnitude)

> Also worth expanding on not every bucket is full (particularly the last bucket). Or TMI?

Beyond the scope of this doc I think.